src/app/public/modules/phone-field/phone-field-adapter.service.ts

```diff
@@ -71,8 +71,8 @@ export class SkyPhoneFieldAdapterService implements OnDestroy {
     element.focus();
   }
 
-  public focusPhoneInput(): void {
-    this.focusElement(document.querySelector('.sky-phone-field-container input') as HTMLElement);
+  public focusPhoneInput(element: HTMLElement): void {
```

Comment:
We should pass in an `ElementRef` to the DOM adapter since the adapter could be replaced with another adapter for a different platform.

Reply:
Good call. This has been done. I also updated all the other methods to do the same.